openrr-base/examples/ros_control_diff_drive.rs

```diff
@@ -0,0 +1,67 @@
+#[cfg(feature = "ros")]
```

Comment:
I feel better moving this into arci-ros/examples/.
What do you think?
I don't want to introduce ROS dependencies in openrr-base if possible.

Reply:
I thought it was more meant to be an openrr-base example than an arci-ros example. But I agree that we don't want to put ROS dependencies in openrr-base and I strongly agree.

So I will move this code to arci-ros.